### ISSUE TITLE
Update Dependencies for Node v4

### ISSUE THREAD

**Issue by ajmueller** [connectivedx/Phoenix#177]
I've noticed a handful of issues with the latest major release of Node and the dependencies being used.  Here's a quick breakdown:

- `gulp` should be updated to at least 3.9
- `gulp-sass` should be updated to at least 2.0.4
- `gulp-autoprefixer` should be updated to at least 3.0.2
- `gulp-combine-media-queries` is completely broken and should be removed; it hasn't been updated in a very long time
- `gulp-minify-css` should be updated to at least 1.2.1; it now uses `clean-css` which includes media query combining, though I've run into issues with it (see [#607](https://github.com/jakubpawlowicz/clean-css/issues/607), on which I commented, and [#645](https://github.com/jakubpawlowicz/clean-css/issues/645)).

I'd put this all in a PR, but I just finished catching all of these issues on one client project and need to hop on a call for another.

**Comment by elseloop** [comment]
Thanks, @ajmueller! I've been wanting to test this and _boom!_ you beat me to it. I'm hoping to get this all taken care of and out by Friday, should you be bored an looking to write up a PR.

![You're the Best](https://whimsyinthejar.files.wordpress.com/2015/06/giphy.gif)

**Comment by ajmueller** [comment]
You just keep knocking it out of the park with these Bill Murray GIFs.

![Nice work](https://s3.amazonaws.com/rapgenius/tumblr_ljaf1lM7Xc1qh8ztro1_500.gif)

**Comment by pc-pdx** [comment]
yesyes. i've just grabed a fresh checkout as of yesterday and had to manually copy [ Assets\src\node_modules\gulp-combine-media-queries\node_modules\through2\node_modules\readable-stream ] over from an existing project in order to complete a build. Seems this a [known, possible race issue](https://github.com/substack/node-browserify/issues/810), FWIW.  
All that aside, i believe i'm still running node 0.10.33 and as such, this readable-stream issue may be a full-stop blocker (?) -  let me know if that warrants a separate ticket as i've found it impossible to build without intervention.
*tl;dr:*
![reading my tickets](https://coedmagazine.files.wordpress.com/2012/08/bill-murray-little-shop-of-horrors.gif)

**Comment by elseloop** [comment]
Fixed in 2.2.0 release
